File: examples/cylinder.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
from __future__ import unicode_literals

from hilma import Mesh, cylinder, savePly

mesh = cylinder(1.0, 2.0, 36, 1, 1, True)
savePly("cylinder.ply", mesh, False)

File: examples/cylinder.cpp
#include <string>
#include <sstream>
#include <unistd.h>
#include <iostream>

#include "hilma/types/Mesh.h"
#include "hilma/ops/generate.h"
#include "hilma/io/ply.h"

int main(int argc, char **argv) {

    hilma::Mesh mesh = hilma::cylinder(1.f, 2.f, 36, 1, 1, true);
    hilma::savePly("cylinder.ply", mesh, false);

    return 1;
}